src/shogun/transfer/domain_adaptation/DomainAdaptationSVM.cpp

```diff
@@ -95,8 +95,8 @@ bool DomainAdaptationSVM::train_machine(std::shared_ptr<Features> data)
 	int32_t num_training_points = get_labels()->get_num_labels();
 	auto labels = binary_labels(get_labels());
 
-	float64_t* lin_term = SG_MALLOC(float64_t, num_training_points);
 
+	SGVector<float64_t>lin_term(num_training_points);
```

Comment:
spaces

Reply:
Where exactly

Reply:
`SGVector<float64_t> lin_term(num_training_points);`
you need a space between the type and the variable name!

Reply:
https://en.cppreference.com/book/intro/variables

Reply:
or even https://en.cppreference.com/w/cpp/language/direct_initialization